data/import/common/US/currency_store.csv

```diff
@@ -1,2 +1,3 @@
 currency_code,store_name,is_default
-USD,US,1
+EUR,US,1
+CHF,US,0
```

Comment:
This and **all** similar changes, the bug that is integrated was about replacing EUR to USD for the US store, but, here we already have USD, why are we integrating smth?

Reply:
please check https://github.com/spryker/suite-nonsplit/pull/8647